docs/source/cpp/getting_started.rst

```diff
@@ -18,28 +18,29 @@
 .. default-domain:: cpp
 .. highlight:: cpp
 
-User Guide
-==========
+Getting Started
+===============
+
+The following articles exist to enable installation, use, and a basic understanding of Arrow.
+Specifically, it contains: an installation and linking guide; documentation of conventions used 
+in the codebase and suggested for users; and tutorials, including: 
+
+* Using Arrow data structures 
+* Using Arrow's file I/O utilities 
+* Using Arrow's efficient computation tools 
+* Utilizing partitioned datasets with Arrow
```

Comment:
I'm thinking the audience here is people who don't yet know much about Arrow, so we should be specific and not assume familiarity. (For example "Using Arrow data structure" -> "What data structures does arrow have?" -> "Building Arrow arrays and tabular structures".)

```suggestion
* Building Arrow arrays and tabular structures 
* Reading and writing Parquet, Arrow, and CSV files
* Executing compute kernels on arrays
* Reading and writing multi-file partitioned datasets
```